packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -589,6 +565,69 @@ linter:
   }
 }
 
+class PackageDependency {
+  Map<String, List<String>> values = <String, List<String>>{};
```

Comment:
This is a map from package paths, to uses of that package (pubspecs or .packages)? Perhaps a comment to stay that.